test/AWS.Deploy.CLI.IntegrationTests/ServerMode/GetApplyOptionSettings.cs

```diff
@@ -28,18 +33,17 @@ public class GetApplyOptionSettings : IDisposable
         private bool _isDisposed;
```

Comment:
It won't let me comment on line 29 - but is `IntegrationTest` in the namespace still accurate?

If this is no longer relying on network calls and runs quickly (does it?), would `UnitTest` make it more likely that a developer working locally runs it?

Reply:
I'd still want to keep it under `IntergationTest` namespace as it is still technically an Integration Test even if it is not accessing the network. Developers could still run individual classes from the project without running the ones that require network calls. There are some tests in there that are similar.